### PR TITLE
Take stride into account when copying data to/from AV frames.

### PR DESCRIPTION
**Pull request by jerzywilczek**
All resolutions (divisible by 2) should work now.

### CODE REVIEW

File: src/rtp_receiver.rs [hunk 1]
```diff
@@ -172,6 +172,20 @@ fn frame_from_av(decoded: &mut Video, pts_offset: &mut Option<i64>) -> Result<Fr
     })
 }
 
+fn copy_plane_from_av(decoded: &Video, plane: usize) -> bytes::Bytes {
+    let mut result = bytes::BytesMut::with_capacity(
+        decoded.plane_width(plane) as usize * decoded.plane_height(plane) as usize,
+    );
```
Comment (WojciechBarczynski) [inline src/rtp_receiver.rs:178]: Use more descriptive variable name

**Review by WojciechBarczynski — APPROVED** [review]
I'm not sure about making this a part of `rtp_receiver.rs` and `rtp_sender.rs`. `RtpReceiver` / `RtpSender` already have quite a lot of responsibilities

**Comment by jerzywilczek** [comment]
> I'm not sure about making this a part of `rtp_receiver.rs` and `rtp_sender.rs`. `RtpReceiver` / `RtpSender` already have quite a lot of responsibilities

I wasn't sure either, but since they already have helpers called respectively `frame_from_av` and `frame_to_av` and also this work is specific to rtp and won't be used for other ways of providing input/output streams, I decided to put it there

**Review by wkozyra95 — APPROVED** [review]

**Review by noituri — APPROVED** [review]
🙌

**Pull request merged**